uat/custom-components/client-java-sdk/src/main/java/com/aws/greengrass/testing/mqtt5/client/exceptions/DiscoverException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt5.client.exceptions;
+
+/**
+ * Client's exception related to discover parts.
+ */
+public class DiscoverException extends ClientException {
```

Comment:
Discovery

Reply:
Fixed in 13f192dc2c5f293be1752b82b975e696a351858e